translations/kubectl/pt_BR/LC_MESSAGES/k8s.po

```diff
@@ -0,0 +1,3319 @@
+"If you specify a directory, kubernetes will\n"
+"    build a set of files in that directory.  By default only dumps things in "
+"the 'kube-system' namespace, but you can\n"
+"    switch to a different namespace with the --namespaces flag, or specify --"
+"all-namespaces to dump all namespaces.\n"
+"\n"
+"    The command also dumps the logs of all of the pods in the cluster, these "
+"logs are dumped into different directories\n"
+"    based on namespace and pod name."
+msgstr ""
+"\n"
+"    Coleta informações do cluster para debugar e diagnosticar problemas do "
+"cluster.  Por padrão, despeja tudo para o\n"
+"    stdout. Você pode, se quiser, especificar um diretório com --output-"
+"directory.  Se especificar o diretório, kubernetes irá\n"
+"    Montar um conjunto de arquivos no diretório.  Por padrão, apenas coleta "
+"informações no namespace ‘kube-system’ , mas você pode\n"
+"    trocar para um namespace diferente com a flag --namespaces, ou "
+"especificar --all-namespaces para todos os namespaces.\n"
+"\n"
+"    O comando também colete os blogs the todos os pode no cluster, estes "
+"logs são salvos em outros diretórios\n"
+"    baseado no namespace e nome do pod."
+
+#: pkg/kubectl/cmd/clusterinfo.go:37
+msgid ""
+"\n"
+"  Display addresses of the master and services with label kubernetes.io/"
+"cluster-service=true\n"
+"  To further debug and diagnose cluster problems, use 'kubectl cluster-info "
+"dump'."
+msgstr ""
+"\n"
+"  Mostra os endereços dos master e serviços com o label kubernetes.io/"
+"cluster-service=true\n"
+"  Para debugar e diagnosticar outros problemas do cluster, utilize ‘kubectl "
+"cluster-info dump’."
+
+#: pkg/kubectl/cmd/create_quota.go:62
+msgid ""
+"A comma-delimited set of quota scopes that must all match each object "
+"tracked by the quota."
+msgstr ""
+"Lista de valores delimitado por vírgulas para um conjunto de escopos de "
+"quota que devem corresponder para cada objeto rastreado pela quota."
+
+#: pkg/kubectl/cmd/create_quota.go:61
+msgid ""
+"A comma-delimited set of resource=quantity pairs that define a hard limit."
+msgstr ""
+"Um comma-delimited ajusta os pares resource=quantity que define um hard "
+"limit."
+
+#: pkg/kubectl/cmd/create_pdb.go:64
+msgid ""
+"A label selector to use for this budget. Only equality-based selector "
+"requirements are supported."
+msgstr ""
+"Um seletor de label a ser usado para este orçamento. Apenas seletores "
+"baseado em igualdade são suportados."
+
+#: pkg/kubectl/cmd/expose.go:104
+msgid ""
+"A label selector to use for this service. Only equality-based selector "
+"requirements are supported. If empty (the default) infer the selector from "
+"the replication controller or replica set.)"
+msgstr ""
+"Um seletor de label para ser utilizado neste serviço. Apenas seletores "
+"baseado em igualdade são suportados. Se vazio (por padrão) o seletor do "
+"replication controller ou replica set será utilizado.)"
```

Comment:
It seems that both the original and the new msgstr have a leading ")" in the end of the string :)

Reply:
in the original it have that, removed, will followup the original string later